Src/GoogleApis.Auth/OAuth2/Web/AuthorizationCodeWebApp.cs

```diff
@@ -102,7 +103,7 @@ public AuthorizationCodeWebApp(IAuthorizationCodeFlow flow, string redirectUri,
 
             // If the stored token is null or it doesn't have a refresh token and the access token is expired, we need 

```

Comment:
Change the comment with this one:
// Check if a new authorization code is needed.